webapp/components/user_settings/user_settings_theme.jsx

```diff
@@ -125,11 +125,6 @@ export default class ThemeSetting extends React.Component {
                 this.originalTheme = Object.assign({}, this.state.theme);
                 this.scrollToTop();
                 this.props.updateSection('');
-            },
-            (err) => {
```

Comment:
As I understood, on success and on error are currently not supported in mattermost-redux; however, rollback is available whenever failure on saving occurred.
But I agree that on success and on error are something that should be added.